src/cli/add.rs

```diff
@@ -55,13 +56,10 @@ pub struct Args {
     /// Don't install the package to the environment, only add the package to the lock-file.
     #[arg(long)]
     pub no_install: bool,
-}
 
-#[derive(Debug, Copy, Clone)]
-pub enum SpecType {
-    Host,
-    Build,
-    Run,
+    /// The platform(s) for which the dependency should be added
+    #[arg(long)]
+    pub platforms: Option<Vec<Platform>>,
```

Comment:
Doesnt 

```suggestion
    pub platforms: Vec<Platform>,
```

just work?